README.md

```diff
@@ -187,6 +187,21 @@ git submodule sync
 git submodule update --init --recursive
 ```
 
+#### Adjust Build Options (Optional)
```

Comment:
Please move this to the end of `Install PyTorch` section below, with a `#####` heading.

Also, please add the following lines here:

You can adjust the configuration of cmake variables optionally, by doing the following. For example, adjusting the pre-detected directories for CuDNN or BLAS can be done with such a step.

Reply:
Done!